File: p80.py
from decimal import *
from math import sqrt

def main():
    getcontext().prec = 102
    tot = 0
    for i in range(2,100):
        k = Decimal(i).sqrt()
        if int(k) != k:
#            k = str(k)
 #           print(k[0:10])
            k = str(k)[0] + str(k)[2:]#[2:]
 #           print(k)
#            print(k[0:10])

            for j in range(100):
                
                tot = tot + int(k[j])
    print(tot)
main()

File: projecteurler-2015/p80.m
tot = 0
for i = 1:2
    k = sqrt(i)
    if floor(k) ~= k
        k = num2str(k,200)
        k = k(3:length(k))
        for j = 1:100
            tot = tot + str2double(k(j));
        end
    end
end